AdminCore.MailClients/SMTP/Interfaces/ISmtpClient.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using AdminCore.DTOs.MailMessage;
+
+namespace AdminCore.MailClients.SMTP.Interfaces
+{
+    public interface ISmtpClient : IDisposable
+    {
+        void Connect(string host, int port, bool useSsl, CancellationToken cancellationToken = new CancellationToken());
```

Comment:
I don't think we need every possible method here. Let's just include what we use.

Reply:
I'm going to delete everything except the functions we use and their `async` counterparts.